gprofiler/main.py

```diff
@@ -230,8 +239,23 @@ def _strip_container_data(collapsed_data):
             lines.append(line[line.find(';') + 1 :])
         return '\n'.join(lines)
 
+    def send_metadata(self):
```

Comment:
`GProfiler` grows too large and this logic really doesn't need to be here. Move to another class that also has access to the `APIClient` object? (and it can be a field of `GProfiler`, or altogether a different class / function that's just invoked once with the `APIClient` as gProfiler boots)

Reply:
Removed in #134